<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Interface Homme Manchots</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Outils</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="1">
        <widget class="QCheckBox" name="balance2">
         <property name="text">
          <string>Balance 2</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="balance1">
         <property name="text">
          <string>Balance 1</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QCheckBox" name="balance3">
         <property name="text">
          <string>Balance 3</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Choisir un intervalle</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Choisir un passage</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="spinBox"/>
       </item>
       <item row="1" column="2">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="5">
        <widget class="QSpinBox" name="spinBox_3">
         <property name="maximum">
          <number>360000</number>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QSpinBox" name="spinBox_2">
         <property name="maximum">
          <number>360000</number>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QCheckBox" name="sommebalance">
         <property name="text">
          <string>Somme des balances</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>Tracer</string>
         </property>
        </widget>
       </item>
       <item row="2" column="4" colspan="2">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Réinitialiser</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3" colspan="3">
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" rowspan="3" colspan="2">
     <layout class="QVBoxLayout" name="plotLayout"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>690</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionOuvrir"/>
    <addaction name="actionEnregistrer"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>Aide</string>
    </property>
    <addaction name="actionA_propos"/>
   </widget>
   <widget class="QMenu" name="menuHistogrammes">
    <property name="title">
     <string>Analyses</string>
    </property>
    <addaction name="actionAfficher"/>
    <addaction name="actionAlgorithme_de_plats"/>
    <addaction name="actionLisser_la_courbe"/>
   </widget>
   <widget class="QMenu" name="menuD_tections">
    <property name="title">
     <string>Détections</string>
    </property>
    <addaction name="actionAffichier"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuHistogrammes"/>
   <addaction name="menuD_tections"/>
   <addaction name="menuAide"/>
  </widget>
  <action name="actionOuvrir">
   <property name="text">
    <string>Ouvrir ...</string>
   </property>
  </action>
  <action name="actionTracer">
   <property name="text">
    <string>Tracer</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionA_propos">
   <property name="text">
    <string>A propos</string>
   </property>
  </action>
  <action name="actionFermer_le_fichier">
   <property name="text">
    <string>Fermer le fichier</string>
   </property>
  </action>
  <action name="actionAfficher">
   <property name="text">
    <string>Histogrammes</string>
   </property>
  </action>
  <action name="actionAlgorithme_de_plats">
   <property name="text">
    <string>Algorithme de plats</string>
   </property>
  </action>
  <action name="actionEnregistrer">
   <property name="text">
    <string>Enregistrer ...</string>
   </property>
  </action>
  <action name="actionLisser_la_courbe">
   <property name="text">
    <string>Lisser la courbe</string>
   </property>
  </action>
  <action name="actionAffichier">
   <property name="text">
    <string>Afficher</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
